tracer/src/Datadog.Trace/Tagging/TraceTagCollection.cs

```diff
@@ -17,21 +17,44 @@ internal class TraceTagCollection
     {
         private List<KeyValuePair<string, string>>? _tags;
         private string? _cachedPropagationHeader;
+        private string? _decisionMakerValue;
```

Comment:
`_samplingMechanismValue` :)

Reply:
done in [cf86494](https://github.com/DataDog/dd-trace-dotnet/pull/4085/commits/cf86494ad39ef502551c6ae34c1b7f2e5a474735)